.github/workflows/build.yaml

```
@@ -51,12 +47,18 @@ jobs:
           key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
           restore-keys: |
             ${{ runner.os }}-yarn-
+      - name: Yarn install
+        working-directory: web
+        run: yarn install --prefer-offline
       - name: Build web static
-        run: make build-frontend
+        working-directory: web
+        run: yarn build
```

What do you think about using the `make`command in CI in order to guarantee both commands of local and CI?.
```suggestion
        run: make web-build
```

---

You are right! I fixed them.